### PR TITLE
Fix seq2seq doc tests

### PR DESCRIPTION
**Pull request by patil-suraj**
# What does this PR do?

This PR fixes the doc tests for se2seq models (BART, mBART, BigBirdPegasus, PLBart).

### CODE REVIEW

File: src/transformers/models/bigbird_pegasus/modeling_bigbird_pegasus.py [hunk 1]
```diff
@@ -51,17 +51,20 @@
 
 _CHECKPOINT_FOR_DOC = "google/bigbird-pegasus-large-arxiv"
 _CONFIG_FOR_DOC = "BigBirdPegasusConfig"
-_TOKENIZER_FOR_DOC = "PegasusTokenizer"
+_TOKENIZER_FOR_DOC = "PegasusTokenizerFast"
```
Comment (ydshieh) [inline src/transformers/models/bigbird_pegasus/modeling_bigbird_pegasus.py:54]: (just a question) any particular reason to use fast tokenizer here, and not in the other 3 models?

File: src/transformers/models/plbart/modeling_plbart.py [hunk 2]
```diff
@@ -1408,10 +1409,11 @@ def __init__(self, config: PLBartConfig, **kwargs):
     @add_start_docstrings_to_model_forward(PLBART_INPUTS_DOCSTRING)
     @add_code_sample_docstrings(
         processor_class=_TOKENIZER_FOR_DOC,
-        checkpoint=_CHECKPOINT_FOR_DOC,
+        checkpoint="hf-internal-testing/tiny-plbart",
```
Comment (ydshieh) [inline src/transformers/models/plbart/modeling_plbart.py:1412]: nit: I would still prefer to use `_CHECKPOINT_FOR_SEQUENCE_CLASSIFICATION`, but feel free to merge without this change.

**Review by ydshieh — APPROVED** [review]
LGTM, thanks for the fix!

Run locally and all tests pass!

**Review by patrickvonplaten — APPROVED** [review]
Feel free to merge!

**Pull request merged**